src/client/components/ResultMessageModalWindow/CelebrationImage.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import emojiConfettiBall from '../../assets/images/icons/emoji-confetti-ball.svg';
+import dizzyFace from '../../assets/images/icons/emoji-dizzy-face.svg';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function CelebrationImage({ correctAnswer }) {
+  if (correctAnswer > 3) {
+    return (
+      <div className="celebration_image-component">
+        <img src={emojiConfettiBall} alt="emoji-confetti-ball.svg" />
+      </div>
+    );
+  }
+  return (
+    <div className="celebration_image-component">
+      <img src={dizzyFace} alt="dizzy-face.png" />
```

Comment:
Are you sure that you are able to see the dizzyFace image here?

Reply:
Yes Priyanka.. it was tested by Sorybook before I push it ... I'm pretty sure ...

Reply:
Actually the problem is not the path it is css, I am sorry but why everything is flex in your component. You should reconsider using flex, it is not helping but disturbing your design.

Reply:
ok thanks...